superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -527,13 +582,22 @@ int RunCopy(BenchArgs *args) {
             return -1;
         }
     }
-    auto end = std::chrono::steady_clock::now();
 
     // Calculate and display bandwidth if no problem
-    double time_in_sec = std::chrono::duration_cast<std::chrono::duration<double>>(end - start).count();
+    float max_time_in_ms = 0;
+    for (int i = 0; i < args->num_subs; i++) {
+        SubBenchArgs &sub = args->subs[i];
+        float time_in_ms = 0;
+        cuda_err = cudaEventElapsedTime(&time_in_ms, sub.start_event, sub.end_event);
```

Comment:
what's the meaning of dma memory copy's cuda time?

Reply:
CUDA events can be used for all CUDA activities on a specific stream, so it's similar to kernels.